ops.yml

```diff
@@ -34,31 +34,14 @@ pipelines:
           - unzip awscli-bundle.zip && ./awscli-bundle/install -b ~/bin/aws
           - export PATH=~/bin:$PATH
           - aws --version
-          # - git clone https://oauth2:$GH_TOKEN@github.com/$ORG/$REPO
-          - git clone https://$GH_USERNAME:$GH_TOKEN@github.com/$GITHUB_ORG/$GITHUB_REPO && cd $GITHUB_REPO
-          # - cd $REPO && ls -asl #only remote
-          # - RAW_BRANCH_LIST=`git branch -r --sort=-committerdate | grep -v HEAD | head -n 20 | sed 's:.*/::' | awk '{print}' ORS="' '"` #only remote
-          # - echo "ux prompt list '$RAW_BRANCH_LIST$REF' --message='Select the repository branch to build' --name='reference'" >> /tmp/ux.sh && chmod 755 /tmp/ux.sh  #only remote
-          # - if [ "${REF}" = "CUSTOM"  ]; then REF=`/tmp/ux.sh`; fi #only remote
-          # - if [ "${REF}" = "CUSTOM" ]; then REF=`ux prompt input --message='Enter the repository branch to build' --name='reference'`; fi && echo "Going to checkout $REF" #only remote
-          # - git fetch && git checkout $REF #only remote
-          # - aws ecr get-login-password --region $AWS_REGION | docker login --username AWS --password-stdin $AWS_ACCOUNT_NUMBER.dkr.ecr.$AWS_REGION.amazonaws.com/$REPO-$STACK_TYPE
-          - echo $DOCKER_REGISTRY | docker login -u $DOCKER_REGISTRY_USER --password-stdin $DOCKER_REGISTRY
-          - docker build -t $DOCKER_REGISTRY/$GITHUB_ORG/$GITHUB_REPO:latest .
-          - docker push $DOCKER_REGISTRY/$GITHUB_ORG/$GITHUB_REPO:latest
-services:
-  - name: sample-expressjs-app:0.1.1
-    description: A sample expressjs service
-    run: node /ops/index.js
-    port: [ '8080:8080' ]
-    domain: ""
-    env:
-      static:
-        - PORT=8080
-    events:
-      - "github:biagotaski/sample-app:pull_request.merged"
-      - "github:biagotaski/sample-app:pull_request.closed"
-    trigger:
-     - build
-     - start
-     - publish
\ No newline at end of file
+          - git clone https://oauth2:$GITHUB_TOKEN@github.com/$ORG/$REPO
+          - cd $REPO && ls -asl #only remote
+          - RAW_BRANCH_LIST=`git branch -r --sort=-committerdate | grep -v HEAD | head -n 20 | sed 's:.*/::' | awk '{print}' ORS="' '"` #only remote
+          - echo "ux prompt list '$RAW_BRANCH_LIST$REF' --message='Select the repository branch to build' --name='reference'" >> /tmp/ux.sh && chmod 755 /tmp/ux.sh #only remote
+          - if [ "${REF}" = "CUSTOM"  ]; then REF=`/tmp/ux.sh`; fi #only remote
+          - if [ "${REF}" = "CUSTOM" ]; then REF=`ux prompt input --message='Enter the repository branch to build' --name='reference'`; fi && echo "Going to checkout $REF" #only remote
+          - git fetch && git checkout $REF #only remote
+          - ls -asl #only locally
```

Comment:
Can we remove this `ls -asl` and the comments for only local / remote? Not sure we need this at all.